### DISCUSSION TITLE
Not able to intall the Perl interpeter.

### DISCUSSION THREAD

**Discussion by shantamcbain** [Camelcade/Perl5-IDEA#2581] [General]
I am having problems seeing the Perl interpreter after installing the Perl plugin. I am using pyCharm community. What do I need to do to get the Perl interpreter visible  in the list and installed?

**Comment by hurricup** [comment]
https://github.com/Camelcade/Perl5-IDEA/wiki/Getting-started may be helpful. May be a bit outdated.
There are also some videos, like https://www.youtube.com/watch?v=fs4EdJikN2c

**Reply by sillBe10** [reply]
Hi, 

I am trying to launch Perl interpreter for a new project, however the intellij IDEA is not recognizing the interpreter. I am able to get the Perl plugin  to work if I launch a project with the python interpreter, and then add the Perl plugin/local interpreter afterward.

Any idea how I can implement the Perl interpreter from launch? The video linked is 5 years old and the IDEA has changed most of the interface since then.

Thanks!

**Reply by hurricup** [reply]
Here is the recent attempt to record the video about how to use the plugin. May be it helps. https://twitter.com/CamelcadeIDE/status/1629706491572764672

**Comment by sillBe10** [comment]
This looks awesome, Thanks so much! I will follow up with my results. 

I am new to Perl and it’s been much easier to learn Perl with the plugin you developed!
